### DISCUSSION TITLE
Diarize ERROR

### DISCUSSION THREAD

**Discussion by jbacchella** [deepgram/community#369] [General help]
Hello, I am testing your application by transcribing voice to text, separating the speakers (diarize). The problem I have is that when I perform the test with CLI it transcribes the text but separates all the voices as SPEAKER[0]. I have tried it with WAV files and MP3 and they do the same.
I copy the command to see where the error is.

```
curl\
  -X POST \
  "https://api.deepgram.com/v1/listen?punctuate=true&model=nova&language=es-419&diarize=true" \
  -H "Authorization: Token $AUTH_TOKEN" \
  --data-binary @zzz.wav
```
Does anyone know what the error is?
Thanks

**Comment by jpvajda** [comment]
@jbacchella  Thanks for asking this question. have you tried running this same request through our [API playground](https://playground.deepgram.com/)? I'd be curious if you have a different experience. 

This would help us isolate the issue to the CLI or API specifically.  Also if you can provide an example of the return response that would be very helpful.

**Comment by jacob-restart** [comment]
Experiencing a similar sounding error where Deepgram's api is identifying 5 separate speakers when (labelled 0-4 in api response) when there are only two speakers on the call.
